Source/WebCore/html/HTMLFormElement.cpp

```diff
@@ -1007,6 +987,13 @@ Vector<Ref<FormListedElement>> HTMLFormElement::copyListedElementsVector() const
     });
 }
 
+Vector<Ref<ValidatedFormListedElement>> HTMLFormElement::copyValidatedListedElementsVector() const
+{
+    return WTF::compactMap(m_listedElements, [] (auto& weakElement) {
```

Comment:
Nit: no space between [] and (.

Reply:
Fixed in [14f374b](https://github.com/WebKit/WebKit/pull/4988/commits/14f374bd68cbc545b68438162b1eaf5cd06bc554), but please note that grep-ing for both`[] (` and `[](` gives ~1450 occurrences. We might consider disregarding those during reviews.